run.sh

```diff
@@ -354,9 +348,21 @@ if [[ ! -f "$PROJECT_ROOT/data/priv_validator_state.json" ]]; then
 fi
 
 if [ "$#" -ne 0 ]; then
-  exec $PREFIX_CMD "$@"
-elif [ -n "$START_CMD" ]; then
-  exec $PREFIX_CMD $START_CMD
+  export START_CMD="$@"
```

Comment:
this break if docker command has an arg that includes a (white)space, e.g. `docker run .... arg1 arg2 "third arg" ...`

Reply:
This might need tweaking a bit further then, but should be an easy enough change